src/config.rs

```diff
@@ -56,37 +61,12 @@ impl Config {
     }
 
     pub fn validate(&self) -> Result<()> {
-        if self.total_size_limit.0 <= self.target_file_size.0 {
-            return Err(box_err!(
-                "Total size limit {:?} less than target file size {:?}",
-                self.total_size_limit,
-                self.target_file_size
-            ));
+        if self.purge_threshold.0 < self.target_file_size.0 {
+            return Err(box_err!("purge_threshold < target_file_size"));
         }
-
-        if self.cache_size_limit.0 < self.target_file_size.0 {
-            return Err(box_err!(
-                "Cache size limit {:?} less than target file size {:?}",
-                self.cache_size_limit,
-                self.target_file_size
-            ));
-        }
-
-        if self.total_size_limit.0 < self.cache_size_limit.0 {
-            return Err(box_err!(
-                "Total size limit {:?} is less than cache limit size {:?}",
-                self.total_size_limit,
-                self.cache_size_limit
-            ));
-        }
-
-        if self.recovery_mode < 0 || self.recovery_mode > 1 {
-            return Err(box_err!(
-                "Unknown recovery mode {} for raftengine",
-                self.recovery_mode
-            ));
+        if self.cache_limit_per_raft.0 > self.cache_limit.0 {
+            return Err(box_err!("cache_limit_per_raft > cache_limit"));
```

Comment:
How about adjusting to the default value?

Reply:
It confuses people. Let's keep it simple?